views/about.lavender

```diff
@@ -24,7 +24,7 @@ body
       .navbar-collapse.collapse
         ul.nav.navbar-nav
           li
-            a(href="/lavender.html", title="Home") Home
+            a(href="/index.html", title="Home") Home
```

Comment:
Does this include the links for the guide? Specifically, the ones to 'via composer', 'via github', etc.